src/test/java/com/trello/boards/NewBoardTest.java

```diff
@@ -0,0 +1,124 @@
+
+        driver.get("https://trello.com/");
+        driver.findElementByLinkText("Log In").click();
+        driver.findElementById("user").sendKeys("fesenko.b@icloud.com");
+        driver.findElementById("password").sendKeys("B0r1sTr3ll0");
+        driver.findElementById("login").click();
+        Waiters.waitSeconds(4);
+        driver.findElementByXPath(".//button[@aria-label= 'Open Boards Menu']//span").click();
+
+        Assert.assertTrue(userBoard.isEnabled(), "The board isn't found");
+
+        action.click(userBoard).build().perform();
+
+        Assert.assertEquals(boardTitle, "Borys_Tech-Stack", "The board title is wrong");
+    }
+
+    @Test
+    public void addNewBoardCheck() {
+        System.setProperty("webdriver.chrome.driver", "C://Users//Office//Downloads//chromedriver_win32 (1)//chromedriver87.exe");
+        ChromeDriver driver = new ChromeDriver();
+        JavascriptExecutor js = (JavascriptExecutor) driver;
+
+        String onBoardTitle = driver.findElementByXPath(".//input[@class= 'board-name-input js-board-name-input']").getAttribute("value");
+
+        driver.get("https://trello.com/");
+        driver.findElementByLinkText("Log In").click();
+        driver.findElementById("user").sendKeys("fesenko.b@icloud.com");
+        driver.findElementById("password").sendKeys("B0r1sTr3ll0");
+        driver.findElementById("login").click();
+        Waiters.waitSeconds(4);
+        driver.findElementByXPath(".//button[@aria-label= 'Open Boards Menu']//span").click();
+
+        driver.findElementByXPath(".//button[@data-test-id='header-boards-menu-create-board']").click();
+        driver.findElementByXPath(".//input[@aria-label='Add board title']").sendKeys(newBoardTitle);
+        driver.findElementByXPath(".//button[@data-test-id='create-board-submit-button']").click();
+
+        Assert.assertEquals(onBoardTitle, newBoardTitle, "The board title is wrong");
+    }
+
+    @Test
+    public void deleteNewBoardCheck() {
+        System.setProperty("webdriver.chrome.driver", "C://Users//Office//Downloads//chromedriver_win32 (1)//chromedriver87.exe");
+        ChromeDriver driver = new ChromeDriver();
+        JavascriptExecutor js = (JavascriptExecutor) driver;
+
+        String confirmationMessage = driver.findElementByXPath(".//div[@class='big-message quiet closed-board']//h1").getText();
+
+        driver.get("https://trello.com/");
+        driver.findElementByLinkText("Log In").click();
+        driver.findElementById("user").sendKeys("fesenko.b@icloud.com");
+        driver.findElementById("password").sendKeys("B0r1sTr3ll0");
+        driver.findElementById("login").click();
+        Waiters.waitSeconds(4);
+        driver.findElementByXPath(".//button[@aria-label= 'Open Boards Menu']//span").click();
+        driver.findElementByXPath(".//a[@title= '" + newBoardTitle + "']").click();
```

Comment:
Tests should not depend on each other.
Make sure you can run any single test without fail.

Reply:
ok